.gitignore

```diff
@@ -1,5 +1,5 @@
-node_modules
-npm-debug.log
-*.pem
-.env
-package-lock.json
+node_modules
+npm-debug.log
+*.pem
+.env
+package-lock.json

```

Comment:
Any idea why this is showing up as a change? Seems to be all whitespace. Wondering if it has something to do with the original project being imported from Glitch.

Reply:
Actually, I am not sure what happened here.
What I remember is ending an end-line, then removing it and repeating it twice and returning it back to beginning. I think this might have happened because of this.

Reply:
We'll leave it like this for now and check again after we import it to Glitch. It's not a problem really, I just want to make sure we know what caused it.